packages/dds/sequence/README.md

```diff
@@ -1,679 +1 @@
-# @fluidframework/sequence
```

Comment:
Is this readme file intentionally deleted, or just a temporary change to test the trigger?

Reply:
Temporary change to test the trigger.